presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.245
+=============
+
+**Highlights**
+==============
+* New `ENUM_KEY` UDF to get the key for an enum value.
+* Configurable query result compression.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix LambdaDefinitionExpression canonicalization to correctly canonicalize Block constant (:issue:`15424`).
```

Comment:
These are technical details.  Give a user facing description instead. Something like:
Fix wrong results issue for queries with multiple lambda expressions that differ only in array/map/row constants that have the same length.

Reply:
Used your description, but added the issue link.